packages/core/admin/admin/src/content-manager/history/components/VersionHeader.tsx

```diff
@@ -23,7 +23,7 @@ export const VersionHeader = ({ headerId }: VersionHeaderProps) => {
   const [{ query }] = useQueryParams<{
     plugins?: Record<string, unknown>;
   }>();
-  const params = useParams<{ collectionType: string; slug: string }>();
+  const params = useParams<{ collectionType: string; slug: string; singleType: string }>();
```

Comment:
params.singleStypes returns => 'single-type, params.collectionType returns => 'collection-types'

![Screenshot 2024-02-22 at 09 19 16](https://github.com/strapi/strapi/assets/26598053/1bed7bc1-36e8-450f-bbc7-0b2b0c94ea0a)

Reply:
ah okay, do you see an advantage in having two variables? We just went for `collectionType` in the CM and discern which string it is 🤔 not that it really matters, just want to understand your thinking 😄

Reply:
🤔 collectionType is undefined on a single-type page.

I believe the values come from here:

https://github.com/strapi/strapi/blob/85ca1e8e529ea27cc2abaac9cb25d969dad99767/packages/core/admin/admin/src/content-manager/routes.tsx#L34-L51

...but let me know if I missed something.

Side note:
I think it could be interesting to have a common variable for both, but I would prefer `:contentType` over `:collectionType`, and personally I would also prefer `:contentTypeUid` over `:slug`.

Reply:
ah I've seen what's happened, your values are coming from

https://github.com/strapi/strapi/blob/85ca1e8e529ea27cc2abaac9cb25d969dad99767/packages/core/admin/admin/src/content-manager/history/routes.tsx#L7-L28

You could roll it into one and reconfigure but it's really up to you if you care enough, it depends how annoying it is having to check different params etc. but I don't think it would be hard... It's a bit strange to be honest that you've set a parameter as the first part of the URL, I would have probably just done `useMatch` to figure it out....

RE. renaming I would love to and i thought about renaming `slug` to be `model` because that's what its referred to on the BE but it would be huge breaking changes for everyone in V5 who looks at the params and it felt a bit unnecessary. I think we could line it up for V6 where hopefully we won't be doing as large breaking changes. I agree its shit naming though 😄

Reply:
pushed an update to use useMatch, lmk if it's what you had in mind.